dashboard/config/scripts/levels/New Adaptations Project.level

```diff
@@ -23,7 +23,7 @@
     "disable_procedure_autopopulate": "false",
     "top_level_procedure_autopopulate": "false",
     "use_modal_function_editor": "true",
-    "hide_share_and_remix": "true",
+    "hide_share_and_remix": "false",
```

Comment:
we should probably document these flags needed for a standalone project level somewhere 🤔

Reply:
Yeah, i think I just had to set "hide_share_and_remix" to false and "is_project_level" to true; the first was configurable through levelbuilder but the second had to be added manually, not sure if there's a way to configure that on level creation?

Reply:
Ah gotcha, sounds good. Just enabled that via levelbuilder.